code/modules/maps/lateload/generic/planets_lateload.dm

```diff
@@ -0,0 +1,271 @@
+from the overmap and *are not* gateway missions please put it in unique_lateload.dm
+
+For overmap things that aren't planets (like stations, ships, asteroid
+belts, etc) that you want to load in please put them in space_lateloads.dm
+
+For gateway missions please put them in gateway_lateload.dm
+
+
+*/
+
+/datum/map_template/planets_lateload
+	allow_duplicates = FALSE
+	var/associated_map_datum
+
+/datum/map_template/planets_lateload/on_map_loaded(z)
+	if(!associated_map_datum || !ispath(associated_map_datum))
+		log_game("Extra z-level [src] has no associated map datum")
+		return
+
+	new associated_map_datum(GLOB.using_map, z)
+
+/datum/map_z_level/planets_lateload
+	z = 0
+
+/datum/map_z_level/planets_lateload/New(var/datum/map/map, mapZ)
+	if(mapZ && !z)
+		z = mapZ
+	return ..(map)
+
+
+//////////////////////////////////////////////
+/// Example Template                       ///
+//////////////////////////////////////////////
+
+/datum/map_template/planets_lateload/example
+	name = "Example Planet"
+	/// ^ The name of your planet. This is what you will need to put in the lateload_z_level list to actually have it be created.
+	/// If it is not on the list it will not be created, simple as that
+
+	desc = "A completely generic and unassuming planet"
+	/// ^ The description of the planet. Is only seen by admins if they manually load in a planet, dont be
+	/// too verbose. For changing what people see in game when they scan it you will need to edit the overmap effect itself
+	/// (overmap effect as of me writing this are in the overmap folder)
+
+	mappath = "_maps/map_levels/192x192/Class_G.dmm"
+	/// ^ The direct file path for your map. Is case sensitive and you need to use /'s not \'s
+	/// Make sure your map file is in a sensible location please for organization sake
+
+	associated_map_datum = /datum/map_z_level/planets_lateload/example
+	/// ^ This should point to the datum down below. Is used ingame after the z level is index'd for debugging purposes
+
+	ztraits = list(ZTRAIT_AWAY = TRUE, ZTRAIT_GRAVITY = TRUE)
+	/// Z level traits. Controls some behavior of the z level. Check out maps.dm for more information
+
+/datum/map_z_level/planets_lateload/example
+	name = "Away Mission - Example Planet"
+	/// Does not have to be the same name as the one you used above. Keep it short and to the point
+
+	base_turf = /turf/space/basic
+	/// What the base turf of the z level is, aka what is left when something like maxcap bomb goes off. If not defined
+	/// it will default to /turf/simulated/space
+
+	flags = MAP_LEVEL_CONTACT|MAP_LEVEL_PLAYER
+	/// Z level flags. Similiar to traits except they control a couple different things. These two are what you are likely going
+	/// to be using a lot. MAP_LEVEL_CONTACT means you will be able to hear station alerts and such. MAP_LEVEL_PLAYER simply tells
+	/// the z level that players will likely be on the z level. Check out maps.dm for more information
```

Comment:
Why the fuck is this coded in, if you want a functional example, I'd recommend putting it in its own file and unticking it.

Reply:
:pain: Yeah I forgot about that, I did that way early on in the project and meant to put it somewhere else. Has been moved to an unticked file